parser/parser.go

```diff
@@ -40,6 +40,9 @@ func Parse(reader io.Reader) ([]Command, error) {
 			command.Args = string(fields[1])
 			// copy command for validation
 			modelCommand = command
+		case "MMPROJ":
```

Comment:
I feel like `MMPROJ` isn't the friendliest term, maybe `MULTIMODAL` instead? Curious what other people's thoughts are.

Reply:
I tend to agree, but this is the propane from llama.cpp so it seemed to make it easier for others to commit to if it maintained the same name... but happy to change if that's the consensus

Reply:
I definitely think it would be really hard for people to grok.

Reply:
So my only aversion to `MULTIMODAL` is that feels like a bool, not a file path. I would also assert that the only people using it are also pulling and quantizing a model for llama.cpp compatibility so it would likely be easier for them to understand.... 

Alternative suggestion might be `MMPROJ_FILE` as it is the MMProj file?